### ISSUE TITLE
Ontology structure

### ISSUE THREAD

**Issue by blancabaselga** [IHTSDO/snomed-owl-toolkit#24]
When I use this tool I get an ontology which structure is not the same as the structure shown, for example, in the ontology repository BioPortal. Is there anything I could do to get the same structure?

**Comment by kaicode** [comment]
Hi @blancabaselga,

It looks like the SNOMED CT structure shown in BioPortal is the inferred structure. The SNOMED OWL Toolkit generates an ontology file containing the stated/asserted modeling.

I just used the latest version of the OWL Toolkit (2.2.0) to convert the Jan 2019 SNOMED CT International Edition and loaded it into Protégé. The asserted view looks quite different to the inferred view. I installed the ELK reasoner which we recommend for reasoning over SNOMED CT, it's much faster than the default one in Protégé. Once ELK had run the structure of the inferred view looks the same as the one shown in BioPortal.

<img width="1164" alt="screenshot 2019-02-13 at 12 25 44" src="https://user-images.githubusercontent.com/2010517/52712149-a4999900-2f8b-11e9-9d5a-bbb6f203d08f.png">

I hope this helps.

Kind regards,
Kai
